.travis.yml

```diff
@@ -24,4 +24,6 @@ matrix:
     env: RAILS_VERSION="~> 4.1.16"
   - rvm: 2.5.1
     env: RAILS_VERSION="~> 4.1.16"
-before_install: gem install bundler -v 1.16.2
+before_install:
+  - gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
+  - gem install bundler -v '< 2'
```

Comment:
https://docs.travis-ci.com/user/languages/ruby/#bundler-20

Alternatively we could drop support for Ruby < 2.3 and use Bundler 2.0?